samples/snippets/snippets_notification_receiver.py

```diff
@@ -31,6 +31,9 @@ def receive_notifications(project_id, subscription_name):
     def callback(message):
         print("Received message")
```

Comment:
May I suggest:

print(f"Received message: {message.data}")

Reply:
That should be better! 👍